cpp/src/Slice/Parser.cpp

```diff
@@ -665,7 +681,7 @@ Slice::DocComment::parseFrom(const ContainedPtr& p, DocLinkFormatter linkFormatt
                 // We've encountered an unknown doc tag.
                 if (line[0] == '@')
                 {
-                    auto unknownTag = line.substr(0, line.find_first_of(" \t:"));
+                    auto unknownTag = line.substr(0, line.find_first_of(" \t"));
```

Comment:
Colons aren't supported after doc-comment tags.
Must of gotten some wires crossed with IceRPC.